DOCUMENTATION.md

```diff
@@ -117,10 +117,10 @@ Once that's done, you can install MkDocs and the required dependencies with
 the following command:
 
 ``` shell
-pip install mkdocs-material==9.0.2 \
-            mkdocs-minify-plugin==0.6.2 --use-pep517 \
+pip install mkdocs-material==9.1.6 \
```

Comment:
Just an idea (not a change request).

Perhaps it would be better to create `website/requirements.txt` file listing the dependencies and recommend users to run `pipenv install`, `pipenv run mkdocs serve` (this will automatically create virtualenv and keep dependencies separated from the rest of the system)? People might have reasons for keeping one of these packages in a different version.

Reply:
`pip install -r requirements.txt`

Yeah, I considered that at some point, then didn't do it for whatever reason. But probably better to do it that way because I actually forgot to update the version numbers in the doco... 😄

I'll change it.